src/main/java/org/codehaus/mojo/license/api/LoadedProjectDependencies.java

```diff
@@ -0,0 +1,28 @@
+
+package org.codehaus.mojo.license.api;
+
+import java.util.Set;
+import org.apache.maven.artifact.Artifact;
+
+public class LoadedProjectDependencies {
+
+    private final Set<Artifact> allDependencies;
+    private final Set<Artifact> directDependencies;
+
+    public LoadedProjectDependencies( Set<Artifact> allDependencies, Set<Artifact> directDependencies )
```

Comment:
I vote for making this class immutable. Either copy the constructor params and wrap them in an Collections.unmodifiableSet or find another way to reach the same effect.

Reply:
Will do.